lonestar/analytics/cpu/triangle-counting/README.md

```diff
@@ -1,5 +1,8 @@
+Triangle counting
+================================================================================
+
 DESCRIPTION 
-===========
+--------------------------------------------------------------------------------
 
 This program counts the number of triangles in a given undirected graph. We 
 implement both node-iterator and edge-iterator algorithms from the following:
```

Comment:
There is a 3rd variant: "orderedCount" with 3-nested loops that performs the best. Please mention that. You can refer GAP:http://gap.cs.berkeley.edu/benchmark.html for this 3rd variant.

Reply:
Is the default variant the fastest one? If not, please make it the default in a separate PR.

Reply:
orderedCount is already the default. (unless it was changed in the last few weeks for whatever reason)